opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -412,6 +412,29 @@ namespace Opm {
         initFullTables(deck, "PVTO", m_pvtoTables);
    }
 
+    const size_t EclipseState::getSaturationFunctionFamily() const{
+
+        bool family1 = !m_sgofTables.empty() && !m_swofTables.empty();
+        bool family2 = !m_swfnTables.empty() && !m_sgfnTables.empty() && !m_swfnTables.empty();
```

Comment:
You repeat one of the queries, furthermore as above, it fails for two-phase runs.